sklearn/feature_extraction/tests/test_feature_hasher.py

```diff
@@ -114,14 +114,20 @@ def test_hash_empty_input():
 
 
 def test_hasher_invalid_input():
+    raw_X = [[], (), iter(range(0))]
+
+    h = FeatureHasher(input_type="gobbledygook")
```

Comment:
```suggestion
    feature_hasher = FeatureHasher(input_type="gobbledygook")
```

Reply:
Thank you @glemaitre for your suggestions. We changed the variable name accordingly.